### PR TITLE
Refactor `<Assisted />` component to use new token structure

### PR DESCRIPTION
**Pull request by JanuarAsprilla**
the structure of the `<Assisted />` component is updated to make it more usable and in accordance with the tokens and other components it needs.

### CODE REVIEW

File: src/Assisted/index.tsx [hunk 1]
```diff
@@ -93,7 +92,7 @@ const Assisted = (props: IAssisted) => {
   const currentStep = steps.find((step) => step?.id === currentStepId);
 
   const currentStepIndex = steps.findIndex(
-    (step) => step?.id === currentStepId,
+    (step) => step?.id === currentStepId
   );
 
   return (
```
Comment (wfercanas) [inline src/Assisted/index.tsx:98]: Button appearance shouldn't be "primary" hardcoded, we must use tokens to set that appearance.

Comment (wfercanas) [inline src/Assisted/index.tsx:98]: Icon appearance shouldn't be hardcoded either

Comment (wfercanas) [inline src/Assisted/index.tsx:98]: Text appearance for title, descriptions, and step number shouldn't be hardcoded.

File: src/Assisted/styles.js [hunk 2]
```diff
@@ -17,8 +17,7 @@ const StyledProgressIndicator = styled.div`
   width: ${({ $arrayLength, $currentStep }) =>
     `${($currentStep / $arrayLength) * 100}%`};
   background: ${({ theme }) =>
-    theme?.color?.surface?.primary?.regular ||
-    inube.color.surface.primary?.regular};
+    theme?.assisted?.step?.color || inube.assisted.step.color};
```
Comment (wfercanas) [inline src/Assisted/styles.js:20]: Instead of `step.color` we should use `bar.color`

File: src/Assisted/styles.js [hunk 3]
```diff
@@ -30,10 +29,9 @@ const StyledStepIndicator = styled.div`
   border-radius: 50%;
   border-width: 2px;
   border-style: solid;
-  margin-bottom: ${inube.spacing.s100};
+  margin-bottom: 8px;
   border-color: ${({ theme }) =>
-    theme?.color?.stroke?.primary?.regular ||
-    inube.color.stroke.primary?.regular};
+    theme?.assisted?.bar?.color || inube.assisted.bar.color};
```
Comment (wfercanas) [inline src/Assisted/styles.js:34]: Instead of `bar.color`, we should use `step.color`

File: src/Assisted/styles.js
Comment (wfercanas) [inline src/Assisted/styles.js:1]: The background of the component is not aligned with the tokens. It shouldn't be fully white.

Storybook:
image

Figma:
image

Excel:
image

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/Assisted/styles.js (hunk 3 above)
Comment (JanuarAsprilla) [reply src/Assisted/styles.js:34]: solved

File: src/Assisted/styles.js (hunk 2 above)
Comment (JanuarAsprilla) [reply src/Assisted/styles.js:20]: solved

File: src/Assisted/styles.js
Comment (JanuarAsprilla) [reply src/Assisted/styles.js:1]: solved

File: src/Assisted/index.tsx (hunk 1 above)
Comment (JanuarAsprilla) [reply src/Assisted/index.tsx:98]: solved

Comment (JanuarAsprilla) [reply src/Assisted/index.tsx:100]: solved

**Pull request merged**